integration/qbft/tests/setup_test.go

```diff
@@ -0,0 +1,79 @@
+package tests
+
+import (
+	"context"
+	"fmt"
+	spectypes "github.com/bloxapp/ssv-spec/types"
+	"github.com/bloxapp/ssv/network"
+	p2pv1 "github.com/bloxapp/ssv/network/p2p"
+	protocolforks "github.com/bloxapp/ssv/protocol/forks"
+	protocolp2p "github.com/bloxapp/ssv/protocol/v2/p2p"
+	"github.com/bloxapp/ssv/protocol/v2/sync/handlers"
+	"github.com/bloxapp/ssv/protocol/v2/types"
+	"github.com/bloxapp/ssv/utils/logex"
+	logging "github.com/ipfs/go-log"
+	"go.uber.org/zap"
+	"go.uber.org/zap/zapcore"
+	"testing"
+)
+
+const (
+	maxSupportedCommittee = 10
+	maxSupportedQuorum    = 7
+)
+
+var sharedData *SharedData
+
+type SharedData struct {
+	Logger *zap.Logger
+	Nodes  map[spectypes.OperatorID]network.P2PNetwork
+}
+
+func GetSharedData(t *testing.T) SharedData { //singleton B-)
+	if sharedData == nil {
+		t.Fatalf("shared data hadn't setuped, try to run test with -test.main flag")
+	}
+
+	return *sharedData
+}
+
+func TestMain(m *testing.M) {
+	_ = logging.SetLogLevelRegex("ssv/.*", "debug") // for debugging TODO: ssv/.* or ssv/*. ?
```

Comment:
why is the returned error being ignored?
```suggestion
	_ = logging.SetLogLevelRegex("ssv/.*", "debug") // for debugging TODO: ssv/.* or ssv/*. ?
```